assets/javascripts/running.js

```diff
@@ -32,13 +32,21 @@ function updateTestStatus(newStatus) {
     $('#running_module').text(newStatus.running);
   
     // If a new module have been started, redraw module list
-    if (testStatus.modlist_initialized == 0 || testStatus.running != newStatus.running) {
+    if (testStatus.running != newStatus.running) {
         testStatus.running = newStatus.running;
-        $.ajax("/tests/" + testStatus.jobid + "/modlist").
-            done(function(modlist) {
-                if (modlist.length > 0) {
-                    updateModuleslist(modlist, testStatus.jobid, testStatus.running);
-                    testStatus.modlist_initialized = 1;
+        $.ajax("/tests/" + testStatus.jobid).
```

Comment:
you're not seriously fetching the full HTML of tests/XXX to then throw it away?

And if /modlist is no longer needed, remove its code.